src/helpers/throw_runtime_error/index.js

```diff
@@ -5,24 +5,24 @@ function errorBuilder(functionName, errorMessage) {
     return `[Passable]: Failed to execute '${functionName}': ${errorMessage}`;
 }
 
-function throwRuntimeError(type: number, ...args: Array<string>) {
+function throwRuntimeError(type: string, ...args: Array<string>) {
     switch (type) {
-        case 0:
+        case '0':
             throw new TypeError(errorBuilder('Passable constructor', `Unexpected ${args[0]}, expected string.`));
-        case 1:
+        case '1':
             throw new Error(errorBuilder('Enforce', `${args[0]} - invalid ${args[1]} value.`));
```

Comment:
An errors object could be better for context. something like:

```js
export const ERRORS = {
  INVALID_FORM_NAME: '0',
  INVALID_ENFORCE_VALUE: '1',
  ......
}
```
And when calling the function:
```js
throwRuntimeError((ERRORS.INVALID_FORM_NAME, typeof name);
throwRuntimeError(ERRORS.INVALID_ENFORCE_VALUE, group, typeof value)
```

You would also get the benefit of being able to change the values without having to change them anywhere you throw an error (see commit `a7419ff - error keys as strings`)

Reply:
Good idea. I added a shared constants file that can expose groups of constants at once -
 currently only exposing errors, but others can be added. Got a little stuck naming all the `passableArgs` errors, but other than that, it looks good.